gnome-initial-setup/gis-driver.c

```diff
@@ -312,6 +316,7 @@ gis_driver_startup (GApplication *app)
 
   priv->main_window = gis_window_new (driver);
   priv->assistant = gis_window_get_assistant (GIS_WINDOW (priv->main_window));
+  priv->music_player = gis_music_player_new ("file://" PKGDATADIR "/fbe.ogg");
```

Comment:
Do we have `fbe.ogg` as part of this PR? It's fine if we don't, as long as the music player gracefully fails.
Also I wonder if we should not put the music into a GResource like we do for everything else